BAC0/scripts/Base.py

```diff
@@ -181,7 +181,7 @@ def startApp(self):
                 self._log.warning("Error opening socket: {}".format(error))
                 raise InitializationError("Error opening socket: {}".format(error))
             self._log.debug("Running")
-        except OSError:
+        except OSError as error:
```

Comment:
This one is simply just fixing a variable assignment bug.

`error` is referenced below, and I think you meant to make this change